### ISSUE TITLE
Fine-tune BERT on CORD-2019 dataset

### ISSUE THREAD

**Issue by andra-pumnea** [deepset-ai/COVID-QA#12]
Fine-tune BERT (or word embeddings?) on CORD-2019 dataset published on [Kaggle](https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge):

`[CORD-19 is a resource of over 29,000 scholarly articles, including over 13,000 with full text, about COVID-19, SARS-CoV-2, and related coronaviruses.]`

The dataset has 2GB. I guess the domain is quite different from the FAQ, as the dataset is made up of scientific papers, but could still be valuable to introduce some substantial vocabulary related to the virus.

**Comment by brandenchan** [comment]
I will look in to this!

**Comment by brandenchan** [comment]
Finetuning bert-base-cased to CORD-19 (30k docs, 6million sentences). Estimating it will take about 8-9 hours if it doesn't crash. You can follow the progress here (https://public-mlflow.deepset.ai/#/experiments/2/runs/ea80e00bb7954c4b964fe80576f3eefe)

GPUs: 4 x v100
batch_size: 64
epochs: 1
learning rate: 2e-5

**Comment by tholor** [comment]
Model is published via hugging face's model hub and can be loaded in FARM, haystack or transformers via `deepset/covid_bert_base` (https://public-mlflow.deepset.ai/#/experiments/55/runs/57074b3298e443b89adc5c0bed8590e4). 

The embeddings of this model perform a bit better for question similarity perform than plain BERT, but are (as expected) still a bit worse than sentence-bert. However, it might be useful as a basis for other tasks on CORD-19 data (e.g. extractive QA, NER ...).
